apps/web/src/app/vault/components/vault-items/vault-cipher-row.component.html

```diff
@@ -151,11 +156,25 @@
       <i class="bwi bwi-fw bwi-file-text" aria-hidden="true"></i>
       {{ "eventLogs" | i18n }}
     </button>
-    <button bitMenuItem (click)="restore()" type="button" *ngIf="cipher.isDeleted">
+    <button
+      bitMenuItem
+      (click)="restore()"
+      type="button"
+      *ngIf="
+        (limitItemDeletion$ | async)
+          ? cipher.isDeleted && cipher.permissions.restore && canDeleteCipher
```

Comment:
Sorry, I gave you the wrong feedback [here](https://github.com/bitwarden/clients/pull/13606#discussion_r1974759225). I missed that this was restoring a cipher, so we definitely should just be checking `cipher.permissions.restore` for the permissions.

```suggestion
          ? cipher.isDeleted && cipher.permissions.restore
```